controllers/storagecluster/cephblockpools.go

```diff
@@ -159,8 +159,14 @@ func (obj *ocsCephBlockPools) ensureCreated(r *StorageClusterReconciler, instanc
 			}
 
 			r.Log.Info("Restoring original CephBlockPool.", "CephBlockPool", klog.KRef(cephBlockPool.Namespace, cephBlockPool.Name))
-			existing.ObjectMeta.OwnerReferences = cephBlockPool.ObjectMeta.OwnerReferences
```

Comment:
is this removal due to both LHS & RHS being same?

Reply:
this was done by mistake, I will revert it. Thanks for pointing it out